packages/generator/README.md

```diff
@@ -151,10 +151,9 @@ Default log level of the SDK is WARN (warning). SDK uses [loglevel](https://www.
 
 To change the loglevel, set the desired level on the SDK logger.
 ```javascript
-import * as CommerceSdk from "commerce-sdk";
-import * as log from "loglevel";
+import { sdkLogger } from "commerce-sdk";
 
-log.getLogger(CommerceSdk.COMMERCE_SDK_LOGGER_KEY).setLevel(log.levels.INFO);
+sdkLogger.setLevel(sdkLogger.levels.INFO);
```

Comment:
Have you tested that this works? (Is levels exported from here?)

Reply:
yes, "log" is the root logger. All the features of root logger are available in named logger as well. updated my unit tests to use the same way.